smoketest/scripts/cli/test_service_monitoring_telegraf.py

```diff
@@ -63,5 +63,23 @@ def test_01_basic_config(self):
         for input in inputs:
             self.assertIn(input, config)
 
+    def test_02_loki(self):
+        label = 'r123'
+        loki_url = 'http://localhost'
+        port = '3100'
+
+        self.cli_set(base_path + ['loki', 'url', loki_url])
+        self.cli_set(base_path + ['loki', 'port', port])
```

Comment:
Please add test code for username and password and also check if the exception is present when only username is specified but no password

Reply:
Done!